internal/manifests/build.go

```diff
@@ -8,10 +8,11 @@ import (
 func BuildAll(opt Options) ([]client.Object, error) {
 	res := make([]client.Object, 0)
 
-	cm, err := LokiConfigMap(opt)
+	cm, sha1, err := LokiConfigMap(opt)
```

Comment:
out of curiosity -- why did you choose to add another return value vs creating a function just to return the sha for a specific configmap section?

Reply:
I think LokiConfigMap is the one that marshals the content into binary format and shoves it deep into the resulting Object. To run it outside of that context would require traversing the configmap (`cm.BinaryData["some unknown key"]`) and _knowing_ where to find that data. I think this fits perfectly within the scope of `LokiConfigMap`.